README.md

```diff
@@ -151,6 +151,29 @@ The script allows the user to specify variables, their units, plotting
 style (stacked vs overlaid), as well as the format of the output image.
 Use `-h` for more information.
 
+
+### Data Compression
+
+The `prmon_compress_output.py` script (Python3) can be used to compress the output file
+while keeping the most relevant information.
+The compression algorithm works as follows, for each separate series: 
+* For processes, threads and GPU counts: only keep the changepoints.
+* For rapidly changing metrics (i.e. the other variables): for any three points A, B, and C, if the linear
+interpolation between A and C passes by B ± *threshold* * *range(metric)*, then B is deleted.
+The percentage threshold is set by the `--precision` parameter.
+
+
+The time index of the final output will be the union of the algorithm outputs of the single 
+time series. Each series will have NA values where a point was deleted at a kept index and, unless otherwise
+specified by the `--skip-interpolate` parameter, will be linearly interpolated to maintain a consistent number of data points
```

Comment:
This one is about the code but when I use `--skip-interpolate` I still see missing entries and not `NA`. Is that expected?

Reply:
OK, I get what you mean now. 

1. Pandas writes NA values as empty strings when outputting to text file. I can change this to whatever you think works best.

2. On the other hand, I completely remove the line (sample at time *t*) if all the metrics were removed at time *t*. I could make it so that we retain the original number of lines and add empty lines in that case, if you think that's more useful.

Reply:
I think that's OK to have empty strings. Pandas seem to correctly import the reduced data frame. I would only clarify the text above and say something like "dropped measurements are stored as empty strings, and when imported in pandas interpreted as NaN". Does that make sense?

Reply:
Sure, I will add this to the README. The idea was anyway what you would re-import them into Pandas for the analysis.